msal/src/main/java/com/microsoft/identity/client/internal/controllers/MSALControllerFactory.java

```diff
@@ -134,13 +136,15 @@ public static boolean brokerEligible(@NonNull final Context applicationContext,
             return false;
         }
 
-        //Do not use broker when the audience is MSA only (personal accounts / consumers tenant alias)
-        AzureActiveDirectoryAuthority azureActiveDirectoryAuthority = (AzureActiveDirectoryAuthority) authority;
+        if (!CommonFlightManager.isFlightEnabled(CommonFlight.SUPPORT_MSA_ACCOUNTS_IN_BROKER)) {
```

Comment:
How is this flight going to be delivered?
Is the idea that individual app team will provide the flight value?

Reply:
The idea here is to just use the default value. I just used the flight code since it was there as common flight manager so I thought perhaps the idea was that at some point we would let app teams control it. 

I can actually get rid of this whole code altogether since it doesn't do much.

Reply:
Yep, app teams (or any consumer of common) can control the flights in common.

For this case I think we should let Broker control it, so we can either remove it from here, or expose canSupportMsaAccountsInBroker from Broker side.

Reply:
We already have the flight in Broker as well. This one here was just for double protection since this helps evaluate the flight early (as opposed to during broker handshake and hence I put it here). If this causes confusion then I can just remove this flight block and just let the flow proceed which would then be handled later during hello handshake.

Reply:
I think better to remove it from here.

Reply:
Addressed. Removed.